src/pages/404.astro

```diff
@@ -0,0 +1,70 @@
+---
+import Layout from "../layouts/Layout.astro";
+import Nav from "../components/Nav.astro";
+import MineCraft from "../components/minecraft/index.jsx";
+import Button from "../components/Button.astro";
+---
+
+<style>
```

Comment:
I'm curious if this slows down page load by initializing Minecraft on our 404 page even if people don't end up using. Could we perhaps make it load only in the event someone clicks play, or link them to another page?

Basically, this is awesome, but I want to make sure we don't slow down a 404 page unless someone wants to play.